File: Prod.RutaDigital.Entidades/Recomendacion/Recomendacion.cs
namespace Prod.RutaDigital.Entidades;

public class Recomendacion : Auditoria
{
    public int id_recomendacion { get; set; }
    public int id_modulo { get; set; }
    public int id_nivel_madurez { get; set; }
    public int orden { get; set; }
    public string? codigo { get; set; }
    public string? descripcion { get; set; }
    public int puntos_produce { get; set; }
    public string? imagen { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/recomendaciones/interfaces/recomendacion.ts
import { RecomendacionDetalle } from './recomendacion-detalle';
import { Recurso } from './recurso';

export interface Recomendacion {
  id_recomendacion: number;
  id_modulo: number;
  id_nivel_madurez: number;
  orden?: number;
  codigo: string;
  descripcion: string;
  puntos_produce: number;
  imagen: string;

  fecha_inicio?: Date;
  calificacion: number;
  concluido: boolean;

  nombre_modulo: string;
  orden_modulo: number;
  nombre_nivel_madurez: string;

  detalle: RecomendacionDetalle[];
  recursos: Recurso[];
}
